privacy-snapshot.json

```diff
@@ -49,5 +49,6 @@
   "authentication.api.cx.metamask.io",
   "oidc.api.cx.metamask.io",
   "price.api.cx.metamask.io",
-  "token.api.cx.metamask.io"
+  "token.api.cx.metamask.io",
+  "sepolia.infura.io"
```

Comment:
updated the privacy snapshot.

Reply:
Vault Test: `yarn test:e2e:single test/e2e/vault-decryption-chrome.spec.js --browser=chrome`, this flow turns off basic functionality so wanted to make sure the new fix/changes worked as expected.

Reply:
I don't think this should be needed. You should be able to remove this change and this PR's build will still pass.